elasticsearch/src/main/java/org/apache/calcite/adapter/elasticsearch/ElasticsearchSchema.java

```diff
@@ -120,7 +120,9 @@ private Set<String> indicesFromElastic() throws IOException {
         throw new IllegalStateException(message);
       }
 
+      Set<String> aliases = Sets.newHashSet(root.findPath("aliases").fieldNames());
       Set<String> indices = Sets.newHashSet(root.fieldNames());
+      indices.addAll(aliases);
```

Comment:
I'm not familiar with es adapter, but is it possible to add a unit test to show that it only works after this change?

Reply:
Sure, I need to understand the current ES integration testing method first, I am only familiar with ES testing based on testcontainers

Reply:
Hi,   I added a method to create an alias and added unit tests

zips_alias is an alias of zips

```
@Test void testAlias() {
    calciteAssert()
        .query("select state, city from elastic.zips_alias");
  }
```